user/managing-os/templates.md

```diff
@@ -158,7 +158,7 @@ No changes in any other directories in TemplateBasedVMs persist in this manner.
 
 (1) Upon creation  
 (2) Following shutdown  
-(3) Including [DisposableVM Template]s
+(3) Including any [DisposableVM Template]
```

Comment:
This strikes my ear as ungrammatical. Is it just me?

Reply:
What we're really saying is:
1. This row applies to TemplateBasedVMs.
2. DisposableVMs are a type of TemplateBasedVM.

Reply:
Changed the wording, as requested.